README.md

```diff
@@ -150,6 +150,7 @@ If you are using Ubuntu on WSL and receive the following message when trying to
 ## Running the App / Verifying Installation
 1. `cd casa/`
 1. Run `bin/setup`
+1. Alter `POSTGRES_USER` and `POSTGRES_PASSWORD` in the newly generated `.env` file.
```

Comment:
The most common flow is that the `.env` file is generate silently, everything works and you are none the wiser.

Sometimes, like this you the default credentials don't work so a user needs to update the `.env`.

Thus, we should highlight the changing `.env` is more of a trouble shooting set and should only need to be done if `bin/setup` fails with a credentials error.

If that does happen the user should also run `bin/setup` again after fixing the `.env` file